components/Badge.svelte

```diff
@@ -14,4 +14,4 @@
   }
 </style>
 
-<span class="dib text-align-center white fs-16 br-50 background {$$props.class}" style="--theme-color: {color}" ><slot /></span>
+<span class="mdc-typography dib text-align-center white fs-16 br-50 background {$$props.class}" style="--theme-color: {color}" ><slot /></span>
```

Comment:
Does this make Badge dependent on MDC? If so, should it move to the mdc directory?

Reply:
mdc-typography is only applied if typography is in the project, so it depends on the consumer. Also, it is not in the mdc catalog so I am still leaning toward not moving it for now.